include/api.php

```diff
@@ -4000,11 +4000,11 @@ function api_fr_photoalbum_update($type)
 		throw new BadRequestException("no new albumname specified");
 	}
 	// check if album is existing
-	if (!DBA::exists('photo', ['uid' => api_user(), 'album' => $album])) {
+	if (!Photo::exists(null, ['uid' => api_user(), 'album' => $album])) {
```

Comment:
This ```null``` looks weird here.

Reply:
It's because it's expecting a `photo` row ID. In this case though it's checking for an album.

Reply:
Like I said below, I would prefer adding the resource-id to the condition. Since we don't have an ```album``` table (yet), I wouldn't vote for an own class for this.

Reply:
Yeah, I saw your other comment later.